libr/bin/bfile.c

```diff
@@ -1,7 +1,10 @@
 /* radare2 - LGPL - Copyright 2009-2018 - pancake, nibble, dso */
 
 #include <r_bin.h>
+#include <r_hash.h>
 #include "i/private.h"
+#include "hash/md5.h"
```

Comment:
use r_hash api which i s generic and works for all the hash algorithsm  at the same time, no need to compute them by separate